File: CS/E3470.Web/CustomSessionUtilityService.cs
using DevExpress.XtraPrinting;
using DevExpress.XtraReports.Service.Native.Services;

namespace E3470.Web {
    public class CustomSessionUtilityService : XRSessionUtilityService {
        protected override void EnableXpsExport(PrintingSystemBase printingSystem) {
        }
    }
}

File: VB/E3470.Web/CustomSessionUtilityService.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraPrinting
Imports DevExpress.XtraReports.Service.Native.Services

Namespace E3470.Web
	Public Class CustomSessionUtilityService
		Inherits XRSessionUtilityService
		Protected Overrides Sub EnableXpsExport(ByVal printingSystem As PrintingSystemBase)
		End Sub
	End Class
End Namespace